04-Protocol-Security.md

```diff
@@ -145,8 +153,11 @@ The following functions will also be referenced:
   - Calls `MixHash(ciphertext)`
   - returns `plaintext`
 
-- `ECDH(k, rk)`: performs an Elliptic-Curve Diffie-Hellman operation using `k`, which is a valid `secp256k1` private key, and `rk`, which is a valid public key
-  - The output is X-coordinate of the resulting EC point
+- `ellswift_ecdh_xonly(k, rk)`: performs an Elliptic-Curve Diffie-Hellman operation using `k`, which is a valid `secp256k1` private key, and `rk`, which is a EllSwift
+encoded public key
```

Comment:
nit: pls don't put this on a new line.

Reply:
I fixed the indentation. I do prefer to keep lines around 80 characters.

Reply:
Yeah, I personally also prefer breaking lines at reasonable length in the plain text. But I was told by someone the other day when writing something in SV2-spec, not to use newline characters in markdown, unless new paragraph should be created. Maybe some interpreters require that. Idn. Just saying.